spark/dl/src/main/scala/com/intel/analytics/bigdl/example/optimizerWrapper/README.md

```diff
@@ -0,0 +1,60 @@
+## Summary
+This example demonstrates how to use DLEstimator to train the LeNet5 model and then apply transformation on returned Model which is compatible with ML pipeline
+Learn more about Spark ML please refer to <http://spark.apache.org/docs/latest/api/scala/org/apache/spark/ml/Pipeline.html>
```

Comment:
Add an extra empty line or these two lines will be displayed in one line in the Readme